<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMUPanel</class>
 <widget class="QWidget" name="IMUPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="imuframe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboAddress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>0x68</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x69</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x6A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0x6B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CS33</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Addr:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>GyroR:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPU6050</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LSM6DS3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Filter:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>AccR:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5" rowspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="comboAccRange">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>±2g (±19,6m/s²)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±4g (±39,2m/s²)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±8g (±78,4m/s²)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±16g (±156,9m/s²)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>I2C-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I2C-2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VSPI</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bus:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="comboGyroRange">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>±125º/s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±250º/s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±500º/s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±1000º/s</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>±2000º/s</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QComboBox" name="comboFilter2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50Hz</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboType</tabstop>
  <tabstop>comboAddress</tabstop>
  <tabstop>comboBus</tabstop>
  <tabstop>comboAccRange</tabstop>
  <tabstop>comboGyroRange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
